extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -288,34 +314,82 @@ export class DataTierApplicationWizard {
 	}
 
 	public async deploy(): Promise<mssql.DacFxResult> {
+		const deployStartTime = new Date().getTime();
 		const service = await this.getService(msSqlProvider);
 		const ownerUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 
-		return await service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
+		const result = await service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
+
+		// Deploy Dacpac: 'Deploy button' clicked in deploy summary page, Reporting the event selection to the telemetry
+		TelemetryReporter.createActionEvent(TelemetryViews.DeployDacpac, 'DeployDacpacOperation')
+			.withAdditionalProperties({
+				'deploymentStatus': result.success.toString(),
+				'error': result.errorMessage,
```

Comment:
Is this guaranteed not to contain PII such as paths?

Reply:
(same for all the other similar events you've added)

Reply:
Working on this comment to find out what kind of errorMessages that we get through Dac services.